plugins/buildClusters/README.md

```diff
@@ -0,0 +1,57 @@
+# Build Clusters Plugin
+> API build clusters plugin for the Screwdriver API
+
+## Usage
+
+### Register plugin
+
+```javascript
+const Hapi = require('hapi');
+const server = new Hapi.Server();
+const buildClustersPlugin = require('./');
+
+server.connection({ port: 3000 });
+
+server.register({
+    register: buildClustersPlugin,
+    options: {}
+}, () => {
+    server.start((err) => {
+        if (err) {
+            throw err;
+        }
+        console.log('Server running at:', server.info.uri);
+    });
+});
+```
+
+### Routes
+
+#### Create a buildCluster
```

Comment:
Small thing but we can probably drop the camel case here and just use `build cluster`

Reply:
Sure!